src/shared/bg_misc.cpp

```diff
@@ -2416,7 +2416,18 @@ weapon_t BG_PrimaryWeapon( int const stats[] )
 	return WP_NONE;
 }
 
-static std::unordered_map<std::string, emoticonData_t, Str::IHash, Str::IEqual> emoticons;
+static emoticonData_t Icon( int codePoint )
+{
+	return { codePoint, {} };
+}
+static std::unordered_map<std::string, emoticonData_t, Str::IHash, Str::IEqual> emoticons {
```

Comment:
Should write a script to generate from this: https://fontawesome.com/v4/cheatsheet/